psi4/src/psi4/libpsi4util/PsiOutStream.cc

```diff
@@ -51,6 +51,8 @@ PsiOutStream::PsiOutStream(std::string fname, std::ios_base::openmode mode) {
         is_cout_ = false;
     }
 
+    // This is sort of big, but vsnprintf does not appear to work correctly on all OS's.
+    // Im looking at YOU CentOS
     buffer_.resize(512000);
```

Comment:
I think this large value was randomly chosen by me when investigating one of your errors in a much older PR.  Does it need to be this large?  Probably not a big deal either way, but figured I'd check